lib/temporal/workflow/context.rb

```diff
@@ -55,6 +55,10 @@ def headers
         metadata.headers
       end
 
+      def search_attributes
```

Comment:
nit: you might want to add a comment telling people about the two ways to put things here.